projects/storefrontapp-e2e-cypress/cypress/integration/vendor/cds/merchandising-carousel.e2e-spec.ts

```diff
@@ -76,7 +75,7 @@ function testPDPPage(productId: string): void {
   );
 }
 
-describe.skip('Merchandsing Carousel', () => {
+describe('Merchandsing Carousel', () => {
```

Comment:
have to keep this until our INFRA person say we're clear to touch the sample datas on dev-17

Reply:
the `skip`

Reply:
Added back in